web/sites/default/config/default/views.view.affinity_groups_with_tag.yml

```diff
@@ -302,7 +355,7 @@ display:
           exclude: false
           alter:
             alter_text: true
-            text: '<div class="affinity-group-flag btn btn-outline-secondary my-2">{{ link_flag }}</div>'
+            text: '<a href="/user/login?destination=/affinity-groups{{ view_node }}" class="btn btn-sm btn-primary"><small>Login to join</small></a>'
```

Comment:
Not sure if this is written correctly. Should the destination still have /affinity-groups in it?

Reply:
This was in the wrong area so I corrected it in the next commit